CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 8.16.0
+  - Aligns with stack major and minor versions, no functional changes [#210](https://github.com/elastic/logstash-filter-elastic_integration/pull/210)
+  - Compatible with Logstash and Elasticsearch 8.16 versions
```

Comment:
"Compatible with" is a bit overloaded, consider breaking it out:

```suggestion
  - Embeds Ingest Node components from Elasticsearch 8.16
  - Compatible with Logstash 8.x
```

Reply:
Hm., compatibility across Logstash 8.x will be difficult since <=LS 8.14 version _embedded_ JDKs do not satisfy the requirement. WDYT if we change the statement to `Compatible with Logstash 8.15+`? Or `Compatible with Logstash 8.x as long as JDK requirement meets, 8.16 is recommended`.

Reply:
```suggestion
  - Embeds Ingest Node components from Elasticsearch 8.16
  - Compatible with Logstash 8.15+
```